### DISCUSSION TITLE
Quarantine Timezone Setting

### DISCUSSION THREAD

**Discussion by pappastech** [modoboa/modoboa#2113] [Q&A] [answered]
I'm sure I've missed something easy, but alas, how do I set the timezone for the Quarantine display? I have the correct timezone set in settings.py, but the Quarantine display seems to still use the original Paris timezone.

Thanks,
Tom

**Comment by tonioo** [comment, marked as answer]
Have you reloaded uwsgi?

**Reply by pappastech** [reply]
That did it...thanks.
